src/app/community/_components/PostCardDetailModal.tsx

```diff
@@ -136,10 +190,33 @@ export default function PostCardDetailModal({ cardId, onClose, isMine }: PostCar
       queryKey: ['postData', cardId],
     });
   };
+
   const handleCloseEditModal = () => {
     setIsEditModalOpen(false);
   };
 
+  const handleClickReport = (e: MouseEvent<HTMLDivElement>) => {
+    e.stopPropagation();
+  };
+
+  const MY_POPOVER_OPTION = [
+    {
+      label: '삭제하기',
+      onClick: handleClickDeleteAlertButon,
+    },
+    {
+      label: '수정하기',
+      onClick: handleClickEditModalButton,
+    },
+  ];
+
+  const OTHERS_POPOVER_OPTION = [
+    {
+      label: '신고하기',
+      onClick: handleClickReport,
+    },
+  ];
```

Comment:
사용이 조금 애매한거같네요...?
```suggestion
const generatePopoverOptions = (isMine: boolean) => {
  if (isMine) {
    return [
      {
        label: '삭제하기',
        onClick: handleClickDeleteAlertButton,
      },
      {
        label: '수정하기',
        onClick: handleClickEditModalButton,
      },
    ];
  } else {
    return [
      {
        label: '신고하기',
        onClick: handleClickReport,
      },
    ];
  }
};
```
상수보다 함수로 선언하셔도 좋을거같습니다